ACCEPTANCE_TESTS.rst

```diff
@@ -0,0 +1,163 @@
+S3 Acceptance Tests
+===================
+
+List of all of the various scenarios that need to be handled in implementing
+a S3 transfer manager.
+
+Upload Tests
+------------
+
+General
+~~~~~~~
+* [x] Upload single nonmultipart file
+* [x] Upload single multipart file
+* [x] Upload multiple nonmultipart files
+* [x] Upload multiple multipart files
+* [x] Failed/cancelled multipart upload is aborted and leaves no orphaned parts
```

Comment:
Is it worth calling out a few sub conditions?

* Any failing upload part results in aborted upload.
* The complete multipart upload failing will result in an aborted upload.
* Not being able to read from the source file (or some other error unrelated to making an API call) results in an aborted upload.